apps/dashboard/public/robots.txt

```diff
@@ -1,6 +1,6 @@
 # *
 User-agent: *
-Allow: /
+Disallow: /
```

Comment:
Revert this change

Reply:
Agreed, something in my agent/editor keeps trying to do change this. Will add it to memory.